.github/workflows/CD.yml

```diff
@@ -0,0 +1,116 @@
+name: CD
+
+on:
+  push:
+    tags:        
+      - '**' 
+
+jobs:
+  validate-tag:
+    name: Validate tag
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v4
+        with:
+          fetch-tags: true
+          fetch-depth: 0
+      
+      - name: Check no jumps in tags
+        run: |
+          # If a previous tag is X.Y, valid tags are only X.(Y+1) or (X+1).0
```

Comment:
So we are assuming that we don't have any patch versions (like `MAJOR.MINOR.PATCH`)? It's okay if so, but we should make this tag style known.

Reply:
Yes you are right, the tag (version) format should be made clear somewhere. 

And yes, I was not considering any patch versioning, reducing the format to `MAJOR.MINOR`, because the project is not too big so I don't think there is the need for a `MAJOR.MINOR.PATCH` versioning format.

Reply:
Made the format clear in [210e00add46f0c74d626e77b540e276a4d8b6609](https://github.com/ACCESS-NRI/replace_landsurface/pull/45/commits/210e00add46f0c74d626e77b540e276a4d8b6609)